docs/sources/configure-server/profile-cli.md

```diff
@@ -0,0 +1,234 @@
+
+### Looking up available profiles on a Pyroscope server
+
+You can use the `profilecli query series` command to look up the available profiles on a Pyroscope server. By default it queries the last hour of data, though this can be controlled with the `--from` and `--to` flags. You can narrow the results down with the `--query` flag. See `profilecli help query series` for more information.
+
+#### Query Series Steps
+
+1. **Specify a Query and a Time Range (optional).**
+
+   - You can provide a label selector using the `--query` flag (e.g., `--query='{service_name="my_application_name"}'`)
+   - You can provide a custom time range using the `--from` and `--to` flags (e.g., `--from="now-3h" --to="now"`)
+
+1. **Construct and execute the Query Series command.**
+
+   - Here's a basic command template:
+     ```bash
+     export PROFILECLI_URL=<pyroscope_server_url>
+     export PROFILECLI_USERNAME=<username>
+     export PROFILECLI_PASSWORD=<password>
+
+     profilecli query series --query=<label_name>=<label_value>
+     ```
+
+   - Modify the placeholders (`<pyroscope_server_url>`, `<username>`, etc.) with your actual values.
+
+   - Example command:
+     ```bash
+     export PROFILECLI_URL=https://profiles-prod-001.grafana.net
+     export PROFILECLI_USERNAME=my_username
+     export PROFILECLI_PASSWORD=my_password
+
+     profilecli query series --query='{service_name="my_application_name"}'
+     ```
+
+   - Example output:
+     ```json
+     {
+         "__name__":"memory",
+         "__period_type__":"space",
+         "__period_unit__":"bytes",
+         "__profile_type__":"memory:inuse_objects:count:space:bytes",
+         "__service_name__":"my_application_name",
+         "__type__":"inuse_objects",
+         "__unit__":"count",
+         "cluster":"eu-west-1",
+         "service_name":"my_application_name"
+      }
+     ```
+
+### Reading a raw profile from a Pyroscope server
+
+You can use the `profilecli query merge` command to retrieve a merged (aggregated) profile from a Pyroscope server. The command merges all samples found in the profile store for the specified query time range. By default it looks for samples within the last hour, though this can be controlled with the `--from` and `--to` flags. The source data can be narrowed down with the `--query` flag, in the same way as with the `series` command.
+
+#### Query Merge Steps
+
+1. **Specify optional flags.**
+
+   - You can provide a label selector using the `--query` flag (e.g., `--query='{service_name="my_application_name"}'`)
+   - You can provide a custom time range using the `--from` and `--to` flags (e.g., `--from="now-3h" --to="now"`)
+   - You can specify the profile type via the `--profile-type` flag. The available profile types are listed in the output of the `profilecli query series` command.
```

Comment:
Makes me think we should make this simpler, with flag like `-mem` or `-cpu` to simplify the usage of this.

We should also most likely show using go how to open and read those merged profiles. I suspect most people don't know how to use pprof.